apps/www/src/registry/default/plate-ui/mention-element.tsx

```diff
@@ -18,6 +19,12 @@ export const MentionElement = withRef<
   const element = useElement<TMentionElement>();
   const selected = useSelected();
   const focused = useFocused();
+  const [isMacEnv, setIsMacEnv] = React.useState(false);
+
+  useEffect(() => {
+    // Avoid ssr hydration mismatch
+    setIsMacEnv(IS_APPLE);
+  }, []);
```

Comment:
Thanks for the PR!

It looks like the effect of this part is to ignore `IS_APPLE` on the first render of the component so as to appease hydration. Can we make this more explicit by changing the state variable to `firstRender`, which defaults to true and is set to false inside the `useEffect`, and then using `!firstRender && IS_APPLE` in the conditional?

Reply:
You can copy paste [this part](https://github.com/udecode/plate/blob/56e0051b064d1af654fb31ac483a1bed198eaa96/apps/www/src/hooks/use-mounted.ts#L4-L8). We'll integrate `useMounted` hook in the installation steps later on.

Reply:
It seems like I can directly use the useMounted hook and IS_APPLE to check the state. The changes are done.